projects/hslayers/src/common/layman/access-rights/layman-access-rights.component.ts

```diff
@@ -1,27 +1,170 @@
-import {Component, Input, ViewRef} from '@angular/core';
+import {Component, Input} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+
+import {lastValueFrom} from 'rxjs';
+
+import {HsEndpoint} from '../../endpoints/endpoint.interface';
+import {HsLanguageService} from '../../../components/language/language.service';
+import {HsLaymanService} from '../../../components/save-map/layman.service';
+import {HsLogService} from './../../log/log.service';
 import {accessRightsModel} from '../../../components/add-data/common/access-rights.model';
 
+type LaymanUser = {
+  username: string; // Username of the user.
+  screen_name: string; // Screen name of the user.
+  given_name: string; // Given name of the user.
+  family_name: string; // Family name of the user
+  middle_name: string; // Middle name of the user
```

Comment:
Could you change these property names to camelCase?

Reply:
Done!